action/action-dsl/src/test/java/com/powsybl/action/dsl/ActionDslLoaderTest.java

```diff
@@ -89,6 +92,38 @@ public void testFixTapDslExtension() {
         assertFalse(phaseTapChanger.isRegulating());
     }
 
+    private static List<DeltaTapData> provideParams() {
+        return Arrays.asList(
+                new DeltaTapData(1, 1, 0, "deltaTap0"),
+                new DeltaTapData(1, 2, 1, "deltaTap1"),
+                new DeltaTapData(1, 3, 2, "deltaTap2"),
+                new DeltaTapData(1, 3, 3, "deltaTap3"),
+                new DeltaTapData(1, 3, 10, "deltaTap10"),
+                new DeltaTapData(1, 0, -1, "deltaTapMinus1"),
+                new DeltaTapData(1, 0, -2, "deltaTapMinus2"),
+                new DeltaTapData(1, 0, -10, "deltaTapMinus10")
+        );
+    }
+
+    @Test
+    public void testDeltaTapDslExtension() {
```

Comment:
Note : maybe we should add a dummy unit test for the behaviour on invalid phase shifter ID, or on transformers which do not have a phase shifter.